src/components/AddPostButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+import { MdAddCircleOutline } from 'react-icons/md';
+import { Button, ButtonGroup } from '@material-tailwind/react';
+
+const AddPostButton = () => {
+  const [isSplit, setIsSplit] = useState(false);
+
+  const handleButtonClick = () => {
+    setIsSplit(!isSplit);
```

Comment:
state 업데이트 해줄 때 이전 상태 의존하여 바꿔주면 좋을 것 같아요!

```suggestion
    setIsSplit((prev) => !prev);
```

Reply:
이거 맨날 이렇게 쓰네요 ㅋㅋ 바꿈